### PR TITLE
add crud operation using springboot, logging remaning testing

### PR DESCRIPTION
**Pull request by badsna**

### CODE REVIEW

File: badsnashrestha/SpringBoot/ProductApplication/src/main/java/com/example/productapplication/model/Product.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+@Getter //to generate getter method for the class field
+@Setter //to generate setter method for  the class field
+@Entity
+// indicates that the  annotated class is a JPA entity and should be persisted to a db using JPA. Creates an corresponding db table based on class field
+@RequiredArgsConstructor //generates the constructor that initalizes all final fields in the class
+@Table(name = "products_info") //to customize db table name, should be snake_case
```
Comment (preetbista) [inline badsnashrestha/SpringBoot/ProductApplication/src/main/java/com/example/productapplication/model/Product.java:14]: table name is singular

File: badsnashrestha/SpringBoot/ProductApplication/src/main/java/com/example/productapplication/model/Product.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    //to define name of the column
+    //should be in snake_case
+    @Column(name = "product_id")
+    //should be camelCase
+    private Integer productId;
+
+    @Column(name = "product_name")
+    private String productName;
+
+    @Column(name = "price")
```
Comment (preetbista) [inline badsnashrestha/SpringBoot/ProductApplication/src/main/java/com/example/productapplication/model/Product.java:40]: no need to mention column if single word exist

**Review by preetbista — COMMENTED** [review]
check this review

**Review by RujanRanjitkar — APPROVED** [review]

**Review by RujanRanjitkar — APPROVED** [review]

**Review by preetbista — APPROVED** [review]
ok

**Review by nelenemo — APPROVED** [review]

**Pull request merged**